docs/eigenda/rollup-guides/integration-client.md

```diff
@@ -0,0 +1,69 @@
+# Disperser Authentication
+
+Rollups and AVSs wishing to integrate with EigenDA should connect with the disperser of the network they wish to write to over the [disperser interface spec](https://github.com/Layr-Labs/eigenda/blob/8ec570b8c2b266fad20ea0af14f0f5d84906c39c/api/proto/disperser/disperser.proto).
+
+## BLS Challenge Authentication
+
+In order to disperse a blob to EigenDA via the Eigen Labs hosted dispeser, clients must be authenticated. Clients that have registered with Eigen Labs using their IP address are automatically authenticated by their IP when making calls to `DisperseBlob()`. Alternatively, clients that have registered their Ethereum address with Eigen Labs may authenticate by BLS signing a challenge bytestring when calling `DisperseBlobAuthenticated()`. This section documents how that authentication process works. You can also find an implementation example [here](https://github.com/Layr-Labs/eigenda/blob/8ec570b8c2b266fad20ea0af14f0f5d84906c39c/disperser/apiserver/server_test.go#L75).
```

Comment:
I think `DisperseBlobAuthenticated` is still in Experimental phase (not well tested). Should we mark it so clearly here?

Reply:
Since we are going to GA soon with production customers, it feels like critical paths like blob dispersal authentication should not involve experimental features. What additional testing do we require to complete for us to feel confident here?

Reply:
To clarify what I mean by this, if I put down on this page that this authentication feature is experimental, the reader's next question is "well what does this mean?". So then if I put down that this means that the feature isn't well tested, then their next question is "well why not?".

Reply:
These are valid questions. But the existence of these questions doesn't mean we should hide the fact the API is not sufficiently tested. It's a good practice to make it clear the life stage of a feature or API -- that will actually earn trust rather than losing trust.

Reply:
Understood and agreed that we should be transparent on principle if we insist on documenting an experimental feature, but I don't think there's any value to the feature in its current state because the whole value prop for customers is that they won't have to do IP based auth. In a conversation with @mooselumph we've come to consensus that we should fully test this auth strategy in order to enable customers to exclusively depend on it for GA launch.